src/components/Socials/styles.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+export const Card = styled.div`
+  display: flex;
+  width: 549px;
+  height: 216px;
+  background-color: #F9F9F9;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 8px;
+`
```

Comment:
```suggestion
  overflow: hidden
`
```

Com essa propriedade você não precisa adicionar o border-radius no ícone 😄 o que evita você ter que trocar isso em dois lugares